app/controllers/application_controller.rb

```diff
@@ -3,14 +3,15 @@
 class ApplicationController < ActionController::Base
   include DeviseTokenAuth::Concerns::SetUserByToken
   include Pundit::Authorization
+  rescue_from StandardError, with: :handle_error_in_json_format
```

Comment:
this rescue is optional, could remove it if preferred

Reply:
lets keep unless there is a reason not to =)